packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/delete/billing-autoRenew-delete.controller.js

```diff
@@ -7,9 +8,21 @@ export default class BillingAutoRenewDeleteCtrl {
     this.atInternet = atInternet;
   }
 
+  $onInit() {
+    this.atInternet.trackClick({
```

Comment:
NitPick: I think it could be done in state `onEnter` hook, but will require some extra code to retrieve `atInternet`/`service` from `injector`.